source/_components/sensor.pvoutput.markdown

```diff
@@ -35,3 +35,30 @@ Configuration variables:
 It's recommended to set `scan_interval:` according to a value greater than 60 seconds. The service only allows 60 requests per hour but the sensor's default is 30 seconds.
 </p>
 
+To format the PVoutput sensor you can you the [template component](/topics/templating/). For example:
+
+```yaml
+sensor:
+  - platform: pvoutput
+    system_id: YOUR_SYSTEM_ID
+    api_key: YOUR_API_KEY
+    scan_interval: 150
+  - platform: template
+    sensors:
+      power_consumption:
+        value_template: '{% if is_state_attr("sensor.pvoutput", "power_consumption", "NaN") %}0{% else %}{{ states.sensor.pvoutput.attributes.power_consumption }}{% endif %}'
```

Comment:
All templates needs to be wrapped with `{% raw %}...{% endraw %}` otherwise they are rendered.